src/20_Components/amp-analytics.html

```diff
@@ -306,5 +309,40 @@
   }
   </script>
   </amp-analytics>
+
+    <!-- ## UI elements appearance tracking -->
+      <!--
+    It is possible to track the appearance of an UI element. Here we have an image which is by default hidden and gets displayed 
+    after an amp-bind event. For simplicity, we are using a button. 
+      -->
+      <div>
+        <button class="ampstart-btn caps m1" on="tap:AMP.setState({hideImageTracking: false})">
```

Comment:
suggestion: use the toggle action `tap:an-image.toggle`. In that case also rename the button to `toggle image`.

Reply:
thanks i updated the sample, however I see the analytics endpoint is called twice when I click on the button

Reply:
Ignore my previous comment...I copied and paste the same code in 2 places, this was why!